### ISSUE TITLE
`a nil value` error when closing floating window

### ISSUE THREAD

**Issue by mj-hd** [obaland/vfiler.vim#26]
I'm using floating filer by following config:
```lua
	config.around = {
		options = {
			auto_resize = true,
			header = true,
			name = 'around',
			blend = 20,
			layout = 'floating',
			border = 'none',
			columns = 'indent,devicons,name',
			keep = true,
		},
		mappings = {
			['<esc>'] = action.quit,
			['<C-s>'] = action.open_by_split,
			['s'] = action.toggle_sort,
		},
	}
```

When I close the floating window, following error is thrown:
```
E5108: Error executing lua ...d/.cache/nvim/vfiler.vim/lua/vfiler/windows/floating.lua:10: BufLeave Autocommands
 for "<buffer=7>": Vim(lua):E5108: Error executing lua /home/mjhd/.cache/nvim/vfiler.vim/lua/vfiler/vfiler.lua:358: attempt to call field '_unregister_events' (a nil value)
stack traceback:
        /home/mjhd/.cache/nvim/vfiler.vim/lua/vfiler/vfiler.lua:358: in function '_unregister_events'
        /home/mjhd/.cache/nvim/vfiler.vim/lua/vfiler/vfiler.lua:329: in function 'wipeout'
        /home/mjhd/.cache/nvim/vfiler.vim/lua/vfiler/config.lua:108: in function 'action'
        /home/mjhd/.cache/nvim/vfiler.vim/lua/vfiler/vfiler.lua:215: in function 'do_action'
        /home/mjhd/.cache/nvim/vfiler.vim/lua/vfiler/vfiler.lua:348: in function 'callback'
        .../mjhd/.cache/nvim/vfiler.vim/lua/vfiler/events/event.lua:30: in function '_handle'
        [string ":lua"]:1: in main chunk
        [C]: in function 'nvim_win_close'
        ...d/.cache/nvim/vfiler.vim/lua/vfiler/windows/floating.lua:10: in function 'close'
        ...d/.cache/nvim/vfiler.vim/lua/vfiler/windows/floating.lua:93: in function 'close'
        /home/mjhd/.cache/nvim/vfiler.vim/lua/vfiler/view.lua:196: in function 'close'
        /home/mjhd/.cache/nvim/vfiler.vim/lua/vfiler/vfiler.lua:260: in function 'quit'
        ...jhd/.cache/nvim/vfiler.vim/lua/vfiler/actions/buffer.lua:16: in function 'action'
        /home/mjhd/.cache/nvim/vfiler.vim/lua/vfiler/vfiler.lua:215: in function 'do_action'
        /home/mjhd/.cache/nvim/vfiler.vim/lua/vfiler/vfiler.lua:196: in function '_do_action'
        [string ":lua"]:1: in main chunk
stack traceback:
        [C]: in function 'nvim_win_close'
        ...d/.cache/nvim/vfiler.vim/lua/vfiler/windows/floating.lua:10: in function 'close'
        ...d/.cache/nvim/vfiler.vim/lua/vfiler/windows/floating.lua:93: in function 'close'
        /home/mjhd/.cache/nvim/vfiler.vim/lua/vfiler/view.lua:196: in function 'close'
        /home/mjhd/.cache/nvim/vfiler.vim/lua/vfiler/vfiler.lua:260: in function 'quit'
        ...jhd/.cache/nvim/vfiler.vim/lua/vfiler/actions/buffer.lua:16: in function 'action'
        /home/mjhd/.cache/nvim/vfiler.vim/lua/vfiler/vfiler.lua:215: in function 'do_action'
        /home/mjhd/.cache/nvim/vfiler.vim/lua/vfiler/vfiler.lua:196: in function '_do_action'
        [string ":lua"]:1: in main chunk
```

OS: Ubuntu 22.04 on WSL2
Neovim:NVIM v0.9.1
vfiler.vim: 0edad19b52bdb1f7097d3091049a5f1c086a3d30

**Comment by mj-hd** [comment]
I was able to avoid this error by checkout version before [this commit](https://github.com/obaland/vfiler.vim/commit/61d49d62614b27427ae67ccd4e26d139a45af31f).

**Comment by obaland** [comment]
Thanks for the report.
I will investigate the cause of the problem based on the information you provided.

**Comment by obaland** [comment]
@mj-hd 
I'm sorry.
It was a simple mistake. I have corrected it in b5f7cee71633b3f3d81c830d1b5d7e68a0ca33f4, please apply it.
Thank you for your continued support.

**Comment by mj-hd** [comment]
@obaland thank you, I've confirmed the problem is fixed with the latest version. I'll close this issue.
